### PR TITLE
Add Solaris support

### PR DESCRIPTION
**Pull request by DimmKirr**

### CODE REVIEW

File: tasks/install/Solaris.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    src: "/tmp/filebeat-{{ filebeat_version }}.pkg"
+    state: present
+
+- name: filebeat | Install Filebeat service
+  shell: (svccfg import /opt/filebeat/etc/filebeat.xml) && (svcs | grep filebeat) > /opt/filebeat/service.created
+  args:
+    creates: /opt/filebeat/service.created
+  when: filebeat_start_at_boot
+
+- name: filebeat | Ensure service is running
```
Comment (torian) [inline tasks/install/Solaris.yml]: The service is enabled on the main task file, so this task is not needed here.

**Comment by torian** [comment]
@kireevco Thanks for your PR. Please take a look at the comment about the service. I'd be happy to merge this if you already tested things out.

File: tasks/install/Solaris.yml (hunk 1 above)
Comment (DimmKirr) [inline tasks/install/Solaris.yml]: Thanks for your review. main.yml is the better location. but:

```
- service:
    name: filebeat
    enabled: yes
  when: filebeat_start_at_boot
```

and 

```
- name: filebeat | Ensure service is running
  service:
    name: filebeat
    state: started
  when: filebeat_start_at_boot
```

to my understanding, are not the same. In main.yml we are just ensuring it is enabled, but not started.

sounds like a good solution would be removing my service section and adding `state: started` to main.yml, if you don't mind.

Also, I've tested my change, already running in production.

(of course a user needs to make sure they have a proper pkg file that they can download)

**Comment by DimmKirr** [comment]
@torian I updated my PR. Please let me know what you think.

**Pull request merged**